packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.jsx

```diff
@@ -0,0 +1,133 @@
+                                }
+                            </li>
+                        </ul>
+                        <Text as='h2' weight='bold' className='contract-type-info__content--glossary-heading'>
+                            {localize('Payout')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'This is equal to the payout per point multiplied by the difference between the final price and the strike price.'
+                            )}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content--glossary-heading'>
+                            {localize('Payout per point')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'This is calculated by Deriv which depends on the strike price and the expiry selected by the client.'
+                            )}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content--glossary-heading'>
+                            {localize('Contract value')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'Price offered by Deriv to buy back the previously purchased contract, which depends, among others, on the current spot price and the remaining duration.'
+                            )}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content--glossary-heading'>
+                            {localize('Cut-off time')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'Exact time at which the contract will expire, which falls at 23:59:59 GMT +0 on the expiry date.'
+                            )}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content--glossary-heading'>
+                            {localize('Expiry date')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'This is the day chose as "End Time" by the client. However, if the client selects a duration above 24 hours, the starting day is based on the date in GMT.'
+                            )}
+                        </Text>
+                    </React.Fragment>
+                );
+                break;
+            default:
+                ContractTypeGlossaryTemplate = <Text as='p'>{localize('Glossary not found.')}</Text>;
+                break;
+        }
+    }
+    return <React.Fragment>{ContractTypeGlossaryTemplate}</React.Fragment>;
```

Comment:
I suppose we don't need <React.Fragment> here, you can write just `return ContractTypeGlossaryTemplate`

Reply:
for some reason it did not like fragment being removed. I had to add it back in